configure.ac

```diff
@@ -236,8 +236,10 @@ os_is_windows=no
 case "$host" in
 	*mingw*)
 		os_is_windows=yes
+		old_LIBS=$LIBS
 		AC_SEARCH_LIBS(__memset_chk, ssp, , mingw_has_memset_chk=no)
 		AC_SEARCH_LIBS(__stack_chk_fail, ssp, , mingw_has_stack_chk_fail=no)
```

Comment:
It would be great if these those lines can somehow be moved downwards to the next
```
case "$host" in
	*mingw*)
```
If that can be done, this old_LIBS workaround is not necessary, and if MinGW at some point decides to move libssp to libgcc, this will not break down.

Reply:
> It would be great if these those lines can somehow be moved downwards to the next

AFAICS that's hard to do, because `stack-smash-protection` and `fortify-source` options
rely on results from that case...esac being not set to no. And the later `-lssp` addition
relies on either `stack-smash-protection` or `fortify-source` option being set to yes. So
double checking seems necessary to me unless I'm missing something. If you have
something better, can you please show me?

> If that can be done, this old_LIBS workaround is not necessary, and if MinGW at some point decides to move libssp to libgcc, this will not break down.